src/js/Features.js

```diff
@@ -130,58 +130,50 @@ Features.prototype.isEnabled = function (featureName) {
 
-    for (var i = 0; i < self._features.length; i++) {
-        var feature_tip_html = '';
         if (self._features[i].haveTip) {
-            feature_tip_html = '<div class="helpicon cf_tip" i18n_title="feature' + self._features[i].name + 'Tip"></div>';
+            feature_tip_html = `<div class="helpicon cf_tip" i18n_title="feature${rawFeatureName}Tip"></div>`;
         }
 
-        var newElements = [];
+        const newElements = [];
+
         if (self._features[i].mode === 'select') {
             if (listElements.length === 0) {
-                newElements.push($('<option class="feature" '
-                    + 'value="-1" '
-                    + 'i18n="featureNone" />'));
+                newElements.push($('<option class="feature" value="-1" i18n="featureNone" />'));
             }
-
-            var newElement = $('<option class="feature" id="feature-'
-                + i
-                + '" name="'
-                + self._features[i].name
-                + '" value="'
-                + self._features[i].bit
-                + '" i18n="feature' + self._features[i].name + '" />');
+            const newElement = $(` <option class="feature" id="feature-${i}" name="${rawFeatureName}" value="${featureBit}" i18n="feature${rawFeatureName}" />`);
 
             newElements.push(newElement);
             listElements.push(newElement);
         } else {
             let featureName = '';
             if (!self._features[i].hideName) {
-                featureName = `<td><div>${self._features[i].name}</div></td>`;
+                featureName = `<td><div>${rawFeatureName}</div></td>`;
             }
 
-            var newElement = $('<tr><td><input class="feature toggle" id="feature-'
-                    + i
-                    + '" name="'
-                    + self._features[i].name
-                    + '" title="'
-                    + self._features[i].name
-                    + '" type="checkbox"/></td><td><div>'
-                    + featureName
-                    + '</div><span class="xs" i18n="feature' + self._features[i].name + '"></span></td>'
-                    + '<td><span class="sm-min" i18n="feature' + self._features[i].name + '"></span>'
-                    + feature_tip_html + '</td></tr>');
+            let element = `<tr><td><input class="feature toggle" id="feature-${i}"`;
+            element += `name="${self._features[i].name}" title="${self._features[i].name}"`;
+            element += `type="checkbox"/></td><td><div>${featureName}</div>`;
+            element += `<span class="xs" i18n="feature${self._features[i].name}"></span></td>`;
+            element += `<td><span class="sm-min" i18n="feature${self._features[i].name}"></span>`;
+            element += `${feature_tip_html}</td></tr>`;
+
+            const newElement = $(element);
 
-            var feature_e = newElement.find('input.feature');
+            const featureElem = newElement.find('input.feature');
```

Comment:
Should this be `featureElement` as well?

Reply:
Yeah to be consistent in naming. And it doesn't bite the other binding (used as function argument on line 206)